src/pages/ShareCodePage.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import {View, ScrollView} from 'react-native';
+import ScreenWrapper from '../components/ScreenWrapper';
+import HeaderWithCloseButton from '../components/HeaderWithCloseButton';
+import Navigation from '../libs/Navigation/Navigation';
+import withLocalize, {withLocalizePropTypes} from '../components/withLocalize';
+import QRShareWithDownload from '../components/QRShare/QRShareWithDownload';
+import compose from '../libs/compose';
+import reportPropTypes from './reportPropTypes';
+import withCurrentUserPersonalDetails, {withCurrentUserPersonalDetailsDefaultProps, withCurrentUserPersonalDetailsPropTypes} from '../components/withCurrentUserPersonalDetails';
+import styles from '../styles/styles';
+import roomAvatar from '../../assets/images/avatars/room.png';
+import * as ReportUtils from '../libs/ReportUtils';
+import MenuItem from '../components/MenuItem';
+import Clipboard from '../libs/Clipboard';
+import * as Expensicons from '../components/Icon/Expensicons';
+import getPlatform from '../libs/getPlatform';
+import CONST from '../CONST';
+
+const propTypes = {
+    /** The report currently being looked at */
+    report: reportPropTypes,
+
+    ...withLocalizePropTypes,
+    ...withCurrentUserPersonalDetailsPropTypes,
+};
+
+const defaultProps = {
+    report: undefined,
+    ...withCurrentUserPersonalDetailsDefaultProps,
+};
+
+// eslint-disable-next-line react/prefer-stateless-function
+class ShareCodePage extends React.Component {
+    qrCodeRef = React.createRef();
+
+    render() {
+        const isReport = this.props.report != null && this.props.report.reportID != null;
+
+        const url = isReport ? `https://new.expensify.com/r/${this.props.report.reportID}` : `https://new.expensify.com/details?login=${this.props.session.email}`;
```

Comment:
```suggestion
        const url = isReport ? `${CONST.NEW_EXPENSIFY_URL}r/${this.props.report.reportID}` : `${CONST.NEW_EXPENSIFY_URL}details?login=${this.props.session.email}`;
```

Reply:
To help with localized testing and we might switch to `https://chat.expensify.com` in the future I think 🤔